[2021-10-16 10:33:02,300] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:30:00+00:00 [queued]>
[2021-10-16 10:33:02,340] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:30:00+00:00 [queued]>
[2021-10-16 10:33:02,340] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 10:33:02,341] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 10:33:02,341] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 10:33:02,363] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 10:30:00+00:00
[2021-10-16 10:33:02,377] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T10:30:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpl5r3xhkp', '--error-file', '/tmp/tmp1du7fkua']
[2021-10-16 10:33:02,378] {standard_task_runner.py:77} INFO - Job 22: Subtask get_gross_year_1
[2021-10-16 10:33:02,372] {standard_task_runner.py:52} INFO - Started process 593 to run task
[2021-10-16 10:33:02,439] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:30:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 10:33:02,515] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T10:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T10:30:00+00:00
[2021-10-16 10:33:02,527] {python.py:152} INFO - Done. Returned value was: ['$376.66 ', '2008\n']
[2021-10-16 10:33:02,561] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T103000, start_date=20211016T103302, end_date=20211016T103302
[2021-10-16 10:33:02,594] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 10:33:02,647] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
